### DISCUSSION TITLE
Documenting Parkosz's letters

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#154] [Show and tell]
Re #27:
https://tug.org/TUGboat/tb43-3/tb135bien-parkosz.html
https://doi.org/10.5281/zenodo.3883862
Thanks!

In Polish:
http://www.poradnikjezykowy.uw.edu.pl/wydania/poradnik_jezykowy.804.2023.05.6-J.Bien.pdf

**Comment by psb1558** [comment]
I missed this earlier. It looks good!

**Reply by jsbien** [reply]
Thanks!
